apps/core/src/hooks/useTimeAgo.ts

```diff
@@ -72,7 +90,8 @@ export const timeAgo = (
     epochMilliSecs: number | null | undefined,
     timeNow?: number | null,
     shortenTimeLabel?: boolean,
-    endLabel = `< 1 sec`
+    endLabel = `< 1 sec`,
+    maxTimeLabelInHours = false
```

Comment:
I would probably do something like `maxTimeUnit` here and make it more extendable but will work as well